versionberg/src/main/groovy/io/github/rockerhieu/versionberg/Git.groovy

```diff
@@ -50,4 +54,16 @@ class Git {
         }
         return 0
     }
+    private static boolean checkInnerGitRepo() {
```

Comment:
Using this method is unnecessary. If `gitDir` is undefined, assign the root project git directory into it, then always use `gitDir` in all git commands.